ecosystem/indexer/src/models/write_set_changes.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{models::transactions::Transaction, schema::write_set_changes};
+use aptos_rest_client::aptos_api_types::WriteSetChange as APIWriteSetChange;
+use serde::Serialize;
+
+#[derive(AsChangeset, Associations, Debug, Identifiable, Insertable, Queryable, Serialize)]
+#[diesel(table_name = "write_set_changes")]
+#[belongs_to(Transaction, foreign_key = "transaction_hash")]
+#[primary_key(transaction_hash, hash)]
+pub struct WriteSetChange {
```

Comment:
nit: `WriteSetChange` seems like a weird name as the `WriteSet` itself reflects the change, so the `Change` seems redundant here. Can we find a better name for this?

Reply:
I can use `WriteSet` here too

Reply:
`WriteSetChange` is taken from the existing model in api/types/transaction.rs :-(